web/src/engine/websites/WebAce.ts

```diff
@@ -1,11 +1,12 @@
-import { Tags } from '../../Tags';
+import { Tags } from '../Tags';
 import icon from './WebAce.webp';
-import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../../providers/MangaPlugin';
-import * as Common from '../decorators/Common';
-import { FetchCSS, FetchJSON, FetchRequest } from '../../FetchProvider';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchJSON, FetchRequest } from '../FetchProvider';
+
 function MangaExtractor(element: HTMLDivElement) {
     const anchor = element.querySelector<HTMLAnchorElement>('a');
-    const id = anchor.pathname.split('comics/')[0];
+    const id = anchor.pathname.match(/(\/\S+\/contents\/\d+\/)/)[1];
```

Comment:
Reconsider the `\S+` as it will also matches `/` slashes.
Other places in this file as well...